lunatrace/bsl/frontend/craco.config.js

```diff
@@ -34,7 +35,15 @@ module.exports = {
     configure: { /* Any webpack configuration options: https://webpack.js.org/configuration */
       resolve: {
         fallback: {
-          process: require.resolve("process/browser")
+          // process: require.resolve("process/browser"),
```

Comment:
Are these changes intentional? If so, can you leave a comment in here as to why they are commented out?